src/components/Card/Card.tsx

```diff
@@ -178,6 +178,9 @@ export default function Card(props: React.PropsWithChildren<CardProps>) {
     mainActionLink,
   } = props;
   const hasImage = imageSrc || imageComponent;
+  const finalImageAspectRatio = imageComponent
```

Comment:
Can you add a console warning if both a `imageComponent` and `imageAspectRatio` are set?  The message should say that `imageAspectRatio` will be ignored in favor of the aspect ratio on `imageComponent`.